chainer/link_hooks/spectral_normalization.py

```diff
@@ -124,7 +124,7 @@ class SpectralNormalization(link_hook.LinkHook):
 
             >>> l = L.Convolution2D(3, 5, 3)
             >>> hook = chainer.link_hooks.SpectralNormalization()
-            >>> l.add_hook(hook)
+            >>> l = l.add_hook(hook)
```

Comment:
This verbose assignment is not recommended to users, while it's valid as a test.

To make it clearer,
```
>>> _ = l.add_hook(hook)
```
or
```
>>> l.add_hook(hook)  # doctest:+ELLIPSIS
...
```
or
```
>>> hook = chainer.link_hooks.SpectralNormalization()
>>> l = L.Convolution2D(3, 5, 3).add_hook(hook)
```

Reply:
> ```
> >>> l.add_hook(hook)  # doctest:+ELLIPSIS
> ...
> ```

Sorry, this doesn't work.

Reply:
OK...